x/perp/v2/keeper/amm.go

```diff
@@ -11,10 +11,11 @@ import (
 	types "github.com/NibiruChain/nibiru/x/perp/v2/types"
 )
 
-// EditPriceMultiplier edits the peg multiplier of an amm pool after making
-// sure there's enough money in the perp EF fund to pay for the repeg. These
-// funds get send to the vault to pay for trader's new net margin.
-func (k Keeper) EditPriceMultiplier(
+// UnsafeShiftPegMultiplier: [Without checking x/sudo permissions] Edits the peg
+// multiplier of an amm pool after making sure there's enough money in the perp
+// EF fund to pay for the repeg. These funds get send to the vault to pay for
+// trader's new net margin.
+func (k Keeper) UnsafeShiftPegMultiplier(
```

Comment:
commented before, I would try to make it private

Reply:
[Ditto](https://github.com/NibiruChain/nibiru/pull/1680#discussion_r1408596611)